playbook.yaml

```diff
@@ -15,6 +15,7 @@ content:
     start_paths: ['docs/*', '!docs/_playbook']
   - url: https://github.com/AxonIQ/axon-synapse.git
     start_paths: ['docs/*', '!docs/_playbook']
+    branches: ['docs-0.6', 'docs-0.7']
```

Comment:
To avoid having to change the playbook every time a new version branch is added, you could use
```
branches: ['docs-*']
```
But see my other comment about the branch naming suggesting dedicated branches for docs.